deploy/osd-user-workload-monitoring/04-dedicated-admins-uwm-am-secret-role.Role.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: dedicated-admins-user-workload-monitoring-manage-am-secret
+  namespace: openshift-user-workload-monitoring
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  resourceNames:
+  - alertmanager-user-workload
+  verbs:
+  - '*'
```

Comment:
Do we want to permit delete?  CMO manages (creates) the secret.  I don't know if on delete CMO will recreate it.  And form what I recall of how the create verb functions you won't actually be able to create the resource when using `resourceNames`.  So I think verbs should be: get, list, watch, update, patch.